src/validator/data_model.py

```
@@ -138,4 +138,71 @@ def validate_part_of_total_floor_area(cls, v):
+        return v
+    @field_validator('surface_type')
+    def validate_surface_type(cls, v):
+        valid_types = {"Wall", "Roof", "Floor", "Window", "Door", "Ceiling", "AirWall", "UndergroundWall", "UndergroundSlab"}
+        if v not in valid_types:
+            raise ValueError(f"Surface Type must be one of {valid_types}.")
+        return v
+    @field_validator('outside_boundary_condition')
+    def validate_outside_boundary_condition(cls, v):
+        valid_conditions = {"Outdoors", "Adiabatic", "Ground", "OtherSideCoefficients", "OtherSideConditionsModel", "Surface"}
+        if v not in valid_conditions:
+            raise ValueError(f"Outside Boundary Condition must be one of {valid_conditions}.")
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)